### PR TITLE
Support split protocol.json

### PR DESCRIPTION
**Pull request by kdzwinel**
In the [first commit](https://github.com/ChromeDevTools/debugger-protocol-viewer/commit/66af91babd63c8f1d1cc73d035d3071723bda296) I've adjusted the `update-protocol-json.sh` file to merge `browser_protocol.json` and `js_protocol.json` together for the tot version and to update `v8/protocol.json` with `js_protocol.json`.

Second commit is just a version bump verifying that everything works correctly.

See https://kdzwinel.github.io/debugger-protocol-viewer/ for a live preview.

### CODE REVIEW

File: update-protocol-json.sh [hunk 1]
```diff
@@ -1,22 +1,19 @@
 #!/bin/bash
 set -x
 
-standalone_path="$HOME/code/devtools-standalone/protocol.json"
```
Comment (kdzwinel) [inline update-protocol-json.sh]: I removed the standalone version support for now - had no idea where protocol files are.

**Comment by paulirish** [comment]
I was expecting the hash/date in here to get updated: 
image

File: update-protocol-json.sh (hunk 1 above)
Comment (paulirish) [inline update-protocol-json.sh]: yup! sg

File: update-protocol-json.sh [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
-local_protocol_path="_data/tot/protocol.json"
+local_tot_protocol_path="_data/tot/protocol.json"
+local_v8_protocol_path="_data/v8/protocol.json"
 
-if [ -s $standalone_path ]; then
-  repo_path=$standalone_path
-elif [ -s $chromium_path ]; then
-  repo_path=$chromium_path
-else
+if ! [ -s $standalone_path ]; then
```
Comment (paulirish) [inline update-protocol-json.sh]: wanna nuke this bit too, then?

**Comment by kdzwinel** [comment]
ah crap, I was probably not on the master branch of chromium when I run the script and got outdated version generated. Let me fix that.

**Comment by paulirish** [comment]
Overall: looks good

File: update-protocol-json.sh (hunk 2 above)
Comment (kdzwinel) [inline update-protocol-json.sh]: It was supposed to check if `browser_protocol_path` exists, my bad. Lets keep it, I'll fix the variable.

**Comment by paulirish** [comment]
nailed it.

**Pull request merged**